MuxExoPlayer/src/r2_11_1/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java

```diff
@@ -28,41 +29,52 @@
 public class MuxStatsExoPlayer extends MuxBaseExoPlayer implements AnalyticsListener,
     Player.EventListener {
 
+  @Deprecated
   public MuxStatsExoPlayer(Context ctx, ExoPlayer player, String playerName,
       CustomerPlayerData customerPlayerData,
       CustomerVideoData customerVideoData) {
-    this(ctx, player, playerName, customerPlayerData, customerVideoData, null, true);
+    this(ctx, player, playerName, customerPlayerData,
+        customerVideoData, null, true);
   }
 
+  @Deprecated
   public MuxStatsExoPlayer(Context ctx, ExoPlayer player, String playerName,
       CustomerPlayerData customerPlayerData,
       CustomerVideoData customerVideoData,
       CustomerViewData customerViewData) {
-    this(ctx, player, playerName, customerPlayerData, customerVideoData, customerViewData, true);
+    this(ctx, player, playerName, customerPlayerData, customerVideoData,
+        customerViewData, true);
   }
 
+  @Deprecated
   public MuxStatsExoPlayer(Context ctx, ExoPlayer player, String playerName,
       CustomerPlayerData customerPlayerData,
       CustomerVideoData customerVideoData,
       boolean sentryEnabled) {
-    this(ctx, player, playerName, customerPlayerData, customerVideoData, null, sentryEnabled);
+    this(ctx, player, playerName, customerPlayerData, customerVideoData,
+        null, sentryEnabled);
   }
 
+  @Deprecated
   public MuxStatsExoPlayer(Context ctx, ExoPlayer player, String playerName,
       CustomerPlayerData customerPlayerData,
       CustomerVideoData customerVideoData,
       CustomerViewData customerViewData, boolean sentryEnabled) {
-    this(ctx, player, playerName, customerPlayerData, customerVideoData, customerViewData,
-        sentryEnabled, new MuxNetworkRequests());
+    this(ctx, player, playerName, new CustomerData(customerPlayerData, customerVideoData,
+        customerViewData), sentryEnabled, new MuxNetworkRequests());
   }
 
   public MuxStatsExoPlayer(Context ctx, ExoPlayer player, String playerName,
```

Comment:
We can't actually remove this signature here, let's deprecate it and add the other one that this became

Reply:
I do not think I have removed any constructor, I have deprecated existing and added a new one here: https://github.com/muxinc/mux-stats-sdk-exoplayer/blob/custom_data_update/MuxExoPlayer/src/r2_13_1/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java#L69
I am not sure I see the issue.